### ISSUE TITLE
[Project] Adapting to patient motion in ultrahigh-field magnetic resonance imaging

### ISSUE THREAD

**Issue by Kopanoglu** [Lewis-Kitchingman/VIC-HACK-2024#8]
**Project title:** Adapting to patient motion in ultrahigh-field magnetic resonance imaging
**Project leader:** Emre Kopanoglu --- email: KopanogluE@cardiff.ac.uk

Patient motion affects data consistency in magnetic resonance imaging (MRI), necessitating rescanning the patient. Therefore, several methods have been developed to track and correct the effects of patient motion. However, these are all applicable at lower MRI field strengths. 

Ultrahigh-field (UHF) MRI offers much better image quality, but at the cost of artificial contrast variations in the image. These contrast variations can be mitigated using patient-tailored radiofrequency pulses. Patient motion, however, invalidates these patient-tailored solutions, as they are inherently tailored to the initial patient position as well. When the patient moves, these radiofrequency pulses need to be redesigned, which takes time making it infeasible, or adapted to the new patient position, which hasn't been done - yet.

The goal of this project is to investigate how patient-tailored radiofrequency pulses should change with patient motion. For this purpose, I will provide radiofrequency pulses designed for various patient positions, making it possible to have paired data for initial patient position and a new patient position (e.g. initial versus rightward 5 mm motion). We will use this dataset and machine learning to try to uncover patterns of change in tailored solutions for consistent patterns of motion (e.g. rightward 5 mm motion). Then, we will train neural networks to automate the adaptation of initial patient-tailored solutions to other patient positions, as the patient moves. 

The project leader has extensive experience with radiofrequency pulse design, MRI physics, Matlab programming; and will provide the data. The ideal participant would have experience with training neural networks such as MLPs, cGANs, or U-NETs, and we need at least one participant with these characteristics, but all participants interested in this project are welcome!

**First task - study design and data preprocessing:** Discuss and decide on what type of neural network to use. Discuss characteristics of radiofrequency pulses and decide on which characteristics to use. 

**Second task - train neural networks and test initial performance:** Train neural networks using the provided dataset. Test on pulses designed for an unseen virtual body model. 

**Third task - further tests to characterise performance and understand limitations:** Shuffle the training & testing datasets in a leave-one-out fashion to understand the effect of training dataset on network performance. 

**Fourth task - different types of motion:** Once network characteristics are determined, train different networks for different motion types (rightward, anteriorward, etc.). Check if networks can be cascaded for arbitrary motion along multiple directions.
